src/lib/components/SetUpPaymentSteps/steps/gnosis-safe/ConfirmValues.svelte

```diff
@@ -21,7 +21,8 @@
   let totalAmount: number = weiToDai(workstream.total);
 
   $: topUpExceedsBalance =
-    daiBalance < utils.parseUnits(topUpAmount.toString()).toBigInt();
+    (daiBalance || BigInt(0)) <
```

Comment:
Wonder if we should be using the `??` operator instead of `||` everywhere?

https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or

Reply:
yes we should for sure! gonna change that